src/main/scala/exercises/04_Options.scala

```diff
@@ -34,7 +34,13 @@ object Options {
    * - does not exist: 					"not existing"
    */
   def roomState(rooms: Map[Int, Option[String]], room: Int): String = {
-    error("Fix me")
+    val roomOpt = rooms.getOrElse(room, Some("not existing"))
```

Comment:
It was not immediately clear to me why the default value is `Some("not existing")` instead of just the string. But I see now why. I think that immediately returning the string would be nicer, but I see that you would have to nest everything in another level of if else.